docs/pool-manager.md

```diff
@@ -0,0 +1,74 @@
+# What is the Pool Manager and what does it do?
+
+👉 A component that provisions and manages vLLM pods within the Dual-Pod (DP) Controller architecture. Can be a Kubernetes native service that maintains a pool of idle pods running just the launcher, pre-labeled for IGW’s InferencePool, and transitions them through some different lifecycle states (idle → active → sleeping → deleted) to ensure fast activation of server-running vLLM pods while minimizing cold-start latency.
```

Comment:
A given launcher Pod will not match the label selector for one InferencePool throughout the Pod's lifetime. The whole point of model switching is that this Pod will run different models at different times. Thus, it needs to have its labels switched along with the model, so that it matches the right InferencePool at the right time.

I presume you mean to allow a sleeping to active transition. We have generally been calling that "awake", in alignment with the vLLM "/sleep" and "/wake_up" calls.

Reply:
> As we discussed during call today, we won't have sleeping vllms in the initial phase

Unfortunately, I was not on that call. I am a bit confused. The rest of this document pretty clearly seems to contemplate sleeping vLLM instances.